packages/fast-data-utilities-react/README.md

```diff
@@ -0,0 +1,57 @@
+# FAST React data utilities
+React data utilities are used to map data to components and perform other actions that may be necessary to pass data to other FAST-DNA libraries for manipulation, data validation or storage.
```

Comment:
what are these other actions?  If they don't yet exist can we omit this sentence?  if they do then I think we should describe them.